src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -734,52 +733,39 @@ BtreeLBAManager::_decref_intermediate(
   extent_len_t len)
 {
   auto c = get_context(t);
-  return with_btree<LBABtree>(
+  return with_btree_ret<LBABtree, update_mapping_ret_bare_t>(
     cache,
     c,
     [c, addr, len](auto &btree) mutable {
     return btree.upper_bound_right(
       c, addr
     ).si_then([&btree, addr, len, c](auto iter) {
-      return seastar::do_with(
-	std::move(iter),
-	[&btree, addr, len, c](auto &iter) {
```

Comment:
Is it confirmed that the lifecycle of `iter` doesn't need to be extended by `do_with()` ?

Reply:
Yes, its references don't cross continuations.